### PR TITLE
NIAD-821 sds client

### PR DESCRIPTION
**Pull request by bartek-sarul**

### CODE REVIEW

**Review by Broom94 — DISMISSED** [review]

**Review by ElliotB-96 — APPROVED** [review]

**Review by ElliotB-96 — DISMISSED** [review]

File: service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ImmunizationObservationStatementMapper.java [hunk 1]
```diff
@@ -241,10 +235,6 @@ private String formatDateTimeType(DateTimeType dateTimeType) {
     }
 
     private String formatShortDate(Date date) {
-        return DATE_TIME_FORMATTER_SHORT
-            .format(date
-                .toInstant()
-                .atZone(ZoneId.of(UK_ZONE_ID))
-                .toLocalDateTime());
+        return new SimpleDateFormat("yyyy-MM-dd").format(date);
```
Comment (Broom94) [inline service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ImmunizationObservationStatementMapper.java:238]: We need to keep UK time zone, i'm fixing the issue in 1060

**Review by Broom94 — APPROVED** [review]

File: service/src/main/resources/application.yml [hunk 2]
```diff
@@ -48,4 +48,10 @@ gp2gp:
     rootCA: ${GP2GP_SPINE_ROOT_CA_CERT:}
     subCA: ${GP2GP_SPINE_SUB_CA_CERT:}
   mhs:
-    url: ${GP2GP_MHS_OUTBOUND_URL:http://localhost:8081/mock-mhs-endpoint}
\ No newline at end of file
+    url: ${GP2GP_MHS_OUTBOUND_URL:http://localhost:8081/mock-mhs-endpoint}
+  sds:
```
Comment (mattd-kainos) [inline service/src/main/resources/application.yml:52]: Can we update the README.md to describe each of these?

Also, what is the purpose of the proxy settings. Does SDS API require a proxy?

File: service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ImmunizationObservationStatementMapper.java (hunk 1 above)
Comment (mattd-kainos) [reply service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ImmunizationObservationStatementMapper.java:238]: This is also the wrong date format for the GP2GP HL7

**Review by mattd-kainos — CHANGES_REQUESTED** [review]
Will finish first thing in the am

**Review by mattd-kainos — APPROVED** [review]

**Review by mattd-kainos — CHANGES_REQUESTED** [review]
README update

File: service/src/main/resources/application.yml (hunk 2 above)
Comment (bartek-sarul) [reply service/src/main/resources/application.yml:52]: Readme updated. Proxy removed

**Review by Broom94 — APPROVED** [review]

**Review by mattd-kainos — APPROVED** [review]

**Pull request merged**